Library/Homebrew/cmd/home.rb

```diff
@@ -8,24 +8,24 @@ module Homebrew
   module_function
 
   def home_args
-    Homebrew::CLI::Parser.new do
-      usage_banner <<~EOS
+    Homebrew::CLI::Parser.new do |p|
+      p.usage_banner <<~EOS
```

Comment:
Can this be done without modifying the code and instead using typing comments?

Reply:
I wonder if we can use `.rbi` files for this...

Reply:
That'd be ideal.